### PR TITLE
authenticate/providers: add onelogin support

### PR DESCRIPTION
**Pull request by desimone**
See:
- https://www.pomerium.io/docs/identity-providers.html#onelogin 

**Checklist**:
- [x] documentation updated 
- [x] ready for review

### CODE REVIEW

File: docs/docs/identity-providers.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Enter a name for the application, select **Web app/API** as the **Application Ty
 
 ![Create application form](./microsoft/azure-create-application.png)
 
-Next you will need to create a key which will be used as the **Client Secret** in Pomeriunm's configuration settings. Click on **Keys** from the **Settings** menu.
+Next you will need to create a key which will be used as the **[Client Secret]** in Pomeriunm's configuration settings. Click on **Keys** from the **Settings** menu.
```
Comment (mbarrien) [inline docs/docs/identity-providers.md]: s/Pomeriunm/Pomerium/

File: docs/docs/identity-providers.md [hunk 2]
```diff
@@ -105,20 +105,20 @@ Navigate to **User Settings** then **Applications** using the left-hand menu.
...
 
 ![Create New Credentials](./gitlab/gitlab-create-application.png)
 
 1.Click **Save Application** to proceed.
 
-Your `Client ID` and `Client Secret` will be displayed:
+Your [Client ID] and [Client Secret] will be displayed:
 
-![Gitlab OAuth Client ID and Secret](./gitlab/gitlab-credentials.png)
+![Gitlab OAuth [Client ID] and Secret](./gitlab/gitlab-credentials.png)
```
Comment (mbarrien) [inline docs/docs/identity-providers.md]: The search and replace also doubled the brackets up inside the alt text, here and in a few other places in this file.

File: docs/docs/identity-providers.md (hunk 1 above)
Comment (desimone) [reply docs/docs/identity-providers.md]: Oh goodness. I suppose it's just a matter of time until I start typing pomeranian.

File: docs/docs/identity-providers.md (hunk 2 above)
Comment (desimone) [reply docs/docs/identity-providers.md]: Hopefully that did the trick. I've pushed an update to the site and a new git commit. Thanks @mbarrien  👍

**Comment by desimone** [comment]
Just saw the review of the previous environmental configuration doc PR. Tried to clean that up a little bit too.

**Pull request merged**